### PR TITLE
#84 add intel data

### LINKED ISSUE #84: Add data for Intel
#### Description

Add a file with data for Intel

https://newsroom.intel.com/news/bob-swan-note-to-intel-employees/

Use `companies-on-blm/examples/add-a-company.md` as a reference for how to add data for a company. If information for a field cannot be found, leave it blank or delete that line.

### PR DESCRIPTION
**Pull request by StephaneRocha**
# Description

Add intel data

Fixes #84

## Type of change

Please delete options that are not relevant.

- [x] Company data addition
- [ ] Bug fix
- [ ] New feature

# How Has This Been Tested?

I compared various new sources and social media posts and the same information repeated

**Test Configuration**:
* Browser: Brave

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings

### CODE REVIEW

File: companies/intel.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    type: Memo
+    raw_text: >
+        Black lives matter. Period.
+        While racism can look very different around the world, one thing
+        that does not look different is that racism of any kind will not
+        be tolerated here at Intel or in our communities.
+  - url: https://twitter.com/intel/status/1267686382849441792
+    date_posted: 2020-06-02
+    type: Twitter
+    raw_text: blackouttuesday, blacklivesmatter
```
Comment (kelseyhuse30) [inline companies/intel.md:18]:
```suggestion
    raw_text: #blackouttuesday, #blacklivesmatter
```

File: companies/intel.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+fortune_rank: 45
+name: Intel
+tags: ["fortune500", "tech", "computers", "artificial intelligence"]
+image: name-in-images-folder.png
```
Comment (kelseyhuse30) [inline companies/intel.md:5]: This should reference an image file.  Would you be able to save the image from their tweet and put it into our images folder?  A name like `intel.png` would be perfect. Then you can reference it here.

**Review by kelseyhuse30 — CHANGES_REQUESTED** [review]
Nice work, @StephaneRocha ! You found everything we were looking for.  Just a few quick updates to make and then it's ready to merge. Thank you so much for contributing.

File: companies/intel.md (hunk 2 above)
Comment (kelseyhuse30) [reply companies/intel.md:5]: I am logging an issue to myself to make the instructions for image more clear :)

Comment (StephaneRocha) [reply companies/intel.md]: I actually just forgot about the image :sweat_smile:
Now I added the tweets image as requested.

File: companies/intel.md (hunk 1 above)
Comment (StephaneRocha) [reply companies/intel.md]: done!

**Comment by StephaneRocha** [comment]
ready for review again :)

File: companies/intel.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 fortune_rank: 45
 name: Intel
 tags: ["fortune500", "tech", "computers", "artificial intelligence"]
-image: name-in-images-folder.png
+image: EZe6DCEWoAEim0h.jpeg
```
Comment (kelseyhuse30) [inline companies/intel.md:5]: Nice! Do you mind renaming the image to `intel.jpeg`?

**Review by kelseyhuse30 — CHANGES_REQUESTED** [review]

File: companies/intel.md (hunk 3 above)
Comment (StephaneRocha) [reply companies/intel.md]: no problem, it's done!

**Review by kelseyhuse30 — APPROVED** [review]
Great work, thank you @StephaneRocha !! 👏🏻 💓

**Comment by kelseyhuse30** [comment]
@StephaneRocha your page is now live :) https://diversitycorp.co/companies/intel/

**Pull request merged**